source/server/overload_manager_impl.cc

```diff
@@ -214,6 +166,72 @@ parseTimerMinimums(const ProtobufWkt::Any& typed_config,
 
 } // namespace
 
+/**
+ * Thread-local copy of the state of each configured overload action.
+ */
+class ThreadLocalOverloadStateImpl : public ThreadLocalOverloadState {
```

Comment:
This class is simply being moved, right? (I'll not look closely at it, in this case :>)